src/containers/Movie/MovieInfo/container.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { genresMovieSelector, movieInfoSelector } from 'Store/movie/selectors'
+import { getMovie as getMovieAction } from 'Store/movie/actions'
+import MovieInfoComponent from './component'
+
+
+class MovieInfo extends Component {
+  componentDidMount() {
+    const { getMovie, movieId } = this.props
+    getMovie(movieId)
+  }
+
+  render() {
+    const { info, genres } = this.props
+    return (info ? (
+      <MovieInfoComponent
+        info={info}
+        genres={genres}
+      />
+    ) : null)
+  }
+}
+
+MovieInfo.propTypes = {
+  info: PropTypes.shape({
+    originalTitle: PropTypes.string,
+    originalName: PropTypes.string,
+    originalLanguage: PropTypes.string,
+    overview: PropTypes.string,
+    releaseDate: PropTypes.string,
+    firstAirDate: PropTypes.string,
+    runtime: PropTypes.number,
+    budget: PropTypes.number,
+    revenue: PropTypes.number
```

Comment:
are all of these keys optional?

Reply:
I think yes

Reply:
we can receive just empty movie object from the API?

Reply:
all fields in the response are optional

Reply:
ok